core-common/src/main/java/org/zalando/nakadi/service/subscription/model/Partition.java

```diff
@@ -134,7 +177,8 @@ public String getEffectiveSession() {
 
     @Override
     public String toString() {
-        return eventType + ":" + partition + "->" + state + ":" + session + "->" + nextSession;
+        return eventType + ":" + partition + "->" + state + ":" +
+                session + "->" + nextSession + ":" + failedCommitsCount + ":" + lastDeadLetterOffset;
```

Comment:
```suggestion
        return eventType + ":" + partition + "->" + state + ":" +
                session + "->" + nextSession + "->" + failedCommitsCount + ":" + lastDeadLetterOffset;
```
It's still cryptic, but more consistent this way ;)